### DISCUSSION TITLE
Working version of ROI.plugin.gurobi - ?

### DISCUSSION THREAD

**Discussion by snestler** [dirkschumacher/ompr#359] [Q&A]
Is anyone aware of a working version of ROI.plugin.gurobi?  I would like to try that for a solver as the ones I have been using (glpk, symphony) aren't working (or working quickly enough) for larger problems.  Thank you.

**Comment by ben-schwen** [comment]
[Gurobi 8](https://github.com/roigrp/ROI.plugin.gurobi/tree/gurobi-8)
[Gurobi 9](https://github.com/roigrp/ROI.plugin.gurobi/tree/gurobi-9)

**Comment by datadrivensupplychain** [comment]
I just installed Gurobi 9.5 on my Windows machine and, after installing the `gurobi` package, this worked:
`remotes:::install_github("roigrp/ROI.plugin.gurobi", INSTALL_opts="--no-multiarch")`

Solving massive problems with a flamethrower, I am
